components/Item/RelatedItems.vue

```diff
@@ -53,7 +53,7 @@
             :to="getItemLink(relatedItem)"
           >
             <v-list-item-avatar>
-              <v-img ref="avatarImg" :src="getImageUrl(relatedItem.Id)" />
+              <blurhash-image v-if="relatedItem" :item="relatedItem" />
```

Comment:
```suggestion
              <blurhash-image :item="relatedItem" />
```
This v-if can be removed